modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomLayouts.testcase

```diff
@@ -21,6 +21,136 @@ definition {
 		}
 	}
 
+	@description = "LPS-168228 - Verify it's possible to add many custom fields in the custom layout"
+	@priority = "4"
+	test CanAddManyFieldsOnLayout {
+		task ("Given an object with 15 fields is created") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 168228",
+				objectName = "CustomObject168228",
+				pluralLabelName = "Custom Objects 168228");
+
+			var n = "0";
+
+			while ("${n}" != "15") {
+				echo("${n}");
+
+				ObjectAdmin.addObjectFieldViaAPI(
+					fieldBusinessType = "Text",
+					fieldLabelName = "Custom Field${n}",
+					fieldName = "customField${n}",
+					fieldType = "String",
+					isRequired = "false",
+					objectName = "CustomObject168228");
+
+				var n = ${n} + 1;
```

Comment:
You are right! However, to keep the standard and better read, use the count **For**.